### PR TITLE
supporting inverse modulation

### PR DESCRIPTION
**Pull request by eMaerthin**
this can be implemented differently, by specific explicitly starting_value and limit_value, but the solution I proposed here has one little advantage - no new arguments are introduced :)

### CODE REVIEW

**Comment by eMaerthin** [comment]
The picture below displays the impact of the change.
- no impact if limit_value <= 1
- rescaling of starting point if limit_value > 1

Zrzut ekranu 2021-08-25 o 18 23 09

**Comment by tozanski** [comment]
Looks good, but it also would be fine to have it as a separate Modulation available to be chosen

needs a push to trigger the CI, maybe some test could be the thing to have a change in this PR

**Comment by eMaerthin** [comment]
> Looks good, but it also would be fine to have it as a separate Modulation available to be chosen
> 
> needs a push to trigger the CI, maybe some test could be the thing to have a change in this PR

I would stay with the same Modulation this time as this is just an extension of the very same function supporting one specific scenario. I will add a test and push changes to trigger the travis.

File: test/test_modulations.jl [hunk 1]
```diff
@@ -0,0 +1,26 @@
+include("../src/enums.jl")
+include("../src/event.jl")
```
Comment (tozanski) [inline test/test_modulations.jl:2]: do not need to include. Everything should be available inside module `MocosSim`

File: test/test_modulations.jl [hunk 2]
```diff
@@ -0,0 +1,26 @@
...
+@testset "InfectionModulations" begin
+  state = MocosSim.SimState(3)
+  params = Missing
```
Comment (tozanski) [inline test/test_modulations.jl:6]: `Missing` is a type, the value is `missing` and it is of type `Missing` hence would not fit to the signature of the function

**Comment by eMaerthin** [comment]
@tozanski can we merge this today?

**Pull request closed without merging**